mlflow/projects/databricks.py

```diff
@@ -157,6 +157,7 @@ def _run_shell_command_job(self, project_uri, command, env_vars, cluster_spec):
                  `Runs Get <https://docs.databricks.com/api/latest/jobs.html#runs-get>`_ API.
         """
         # Make jobs API request to launch run.
+        extra_libraries = cluster_spec.pop("libraries", [])
```

Comment:
One caveat is that Python libraries will actually not be honored (since we run the project within a conda environment on the Databricks cluster). We should be sure to at least document this. It'd also be nice to scan through the libraries & warn the user that PyPI libraries will be ignored, e.g:

```
pypi_libs = [lib["pypi"]["package"] for lib in extra_libraries if "pypi" in lib]
if len(pypi_libs) > 0:
  warnings.warn("PyPI libraries %s, specified as part of the Databricks backend config, will be ignored during project execution. Please specify PyPI libraries in your project's conda environment file, as described in https://www.mlflow.org/docs/latest/projects.html#specifying-an-environment" % ", ".join(pypi_libs))